src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -264,38 +283,44 @@ const PreviewHeader = ({
       [readOnlyAccess, () => h(HeaderButton, { onClick: () => setExportingAnalysis(true) }, [
         makeMenuIcon('export'), 'Copy to another workspace'
       ])],
-      // Top level decision on whether user should be prompted to create a runtime
-      [!runtime || currentRuntimeTool !== toolLabel,
+      [!runtime, () => createNewRuntimeOpenIcon],
+      [runtimeStatus === 'Stopped', () => h(HeaderButton, {
+        onClick: () => startAndRefresh(refreshRuntimes, runtime)
+      }, openMenuIcon)],
+      [currentRuntimeTool === tools.Azure.label && _.includes(runtimeStatus, usableStatuses) && currentFileToolLabel === tools.Jupyter.label,
```

Comment:
Why does `runtimeStatus` not need to be checked for Jupyter?

Reply:
So the launchers in general are a travesty of complexity that did not scale well. The `PreviewHeader` component, where this logic lives, is not rendered if `_.includes(status, usableStatuses) && currentRuntimeTool === 'Jupyter'`, and the `AnalysisEditorFrame` is rendered instead